### PR TITLE
feat: include Angular 2

### PR DESCRIPTION
**Pull request by gdi2290**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -104,7 +104,8 @@ If you want to create your own list, please read the [create list instructions](
 - [React](https://github.com/enaqx/awesome-react)
 - [Web Components](https://github.com/mateusortiz/webcomponents-the-right-way)
 - [Polymer](https://github.com/Granze/awesome-polymer)
-- [AngularJS](https://github.com/gianarb/awesome-angularjs)
+- [Angular 2.x](https://github.com/angular-class/awesome-angular2)
```
Comment (sindresorhus) [inline readme.md]: `Angular 2`

File: readme.md [hunk 2]
```diff
@@ -104,7 +104,8 @@ If you want to create your own list, please read the [create list instructions](
...
+- [AngularJS 1.x](https://github.com/gianarb/awesome-angularjs)
```
Comment (sindresorhus) [inline readme.md]: `Angular`

**Comment by sindresorhus** [comment]
You have also adressed my feedback: https://github.com/sindresorhus/awesome/pull/327#issuecomment-142240111

**Comment by gdi2290** [comment]
yeah, sorry I did it for namespacing but it does look a lot better without the "Angular 2 " prefix. The changes are done @sindresorhus

**Pull request closed without merging**